src/client/mod.rs

```diff
@@ -344,9 +344,9 @@ pub struct ListResponse {
     pub keys: Vec<String>,
 }
 
-/// Options that we use when renewing leases on tokens and secrets.
+/// Options that we use when renewing tokens.
 #[derive(Deserialize, Serialize, Debug)]
-struct RenewOptions {
+struct RenewTokenOptions {
```

Comment:
Note this commit doesn't build without the following commit. Git comes up with a real messy diff if I combine them.